k8s.io/configmap-nginx.yaml

```diff
@@ -90,6 +90,18 @@ data:
         rewrite ^/(.*)?$    https://github.com/kubernetes/kubernetes/tree/master/$1 redirect;
       }
       server {
+        server_name dl.k8s.io dl.kubernetes.io;
+        listen 80;
+        listen 443 ssl;
+
+        location / {
+          # Don't require /release/ if you want to get at the Kubernetes release artifacts, the common case.
+          rewrite ^/(v[0-9]+\.[0-9]+.*)$   https://storage.googleapis.com/kubernetes-release/release/$1 redirect;
```

Comment:
re: examples, my point was to catch `/v1.4/foo` but not `v1.4.foo` - but now I am unclear.  Is the path supposed to have `v1.4` or `v.1.4.0`?  The example above show 3 digits, but your test shows 2 digits